docs(chore) revision on calc-constant doc (#1)
ramiy/content: files/en-us/web/css/calc-constant/index.md

{{CSSRef}}

The **`<calc-constant>`** [CSS](/en-US/docs/Web/CSS) [data type](/en-US/docs/Web/CSS/CSS_Types) represents common constants such as `e` and `pi`.
These constants are supported for convenience in CSS so that authors do not have to manually write or calculate them.
Mathematical constants can only be used inside [CSS math functions](/en-US/docs/Web/CSS/CSS_Functions#math_functions) for calculations.
If used outside of a calculation, they're treated like any other keyword.
For example:

> - `animation-name: pi;` refers to an animation named "pi", not the `pi` numeric constant.

Reviewer: I'm happy with this, ish, although I like the practice of having a very short summary at the start:

> The **`<calc-constant>`** [CSS](/en-US/docs/Web/CSS) [data type](/en-US/docs/Web/CSS/CSS_Types) represents common constants such as `e` and `pi`.

...and then putting the other stuff in an H2 Description that goes after H2 Syntax.

Author: That makes sense, I'm happy with either

Reviewer 2: > I like the practice of having a very short summary at the start

Me too.

- `line-height: e;` is invalid, but `line-height: calc(e);` is valid.

> Using `infinity` or `-infinity` is intended to be more convenient for authors to create large fake rather than calculating or generating them through other means.

Reviewer: large fake? Not sure what this means :).

Author: 😂 I meant to write something along the lines of "large dummy / mock data". I'll have another go...

> Although `infinity` and `NaN` are not technically numbers, they are evaluated as {{cssxref("number")}} values, so an expression like `calc(infinity * 1px)` can be used to create an infinite {{cssxref("length")}} value, for example.

Reviewer: I think the bit about "they are evaluated as {{cssxref("number")}} values, so an expression like `calc(infinity * 1px)` can be used to create an infinite {{cssxref("length")}} value, for example" is true about all these constants, so it is misleading to hitch it to "Although `infinity` and `NaN` are not technically numbers".

I don't really think "Although `infinity` and `NaN` are not technically numbers" is helpful. What does "not technically numbers" mean?

```suggestion
In math functions, `<calc-constant>` values are evaluated as {{cssxref("number")}} values, so an expression like `calc(infinity * 1px)` can be used to create an infinite {{cssxref("length")}} value, for example.
```

`NaN` is case-sensitive, but other CSS constants are case-insensitive (e.g., `calc(pi)`, `calc(E)` and `calc(InFiNiTy)` are valid).

## Syntax

The `<calc-constant>` type defines numeric constants that can be used in [CSS math functions](/en-US/docs/Web/CSS/CSS_Functions#math_functions).

### Values

- `e`

  - : The base of the natural logarithm, approximately equal to `2.7182818284590452354`.

- `pi`

  - : The ratio of a circle's circumference to its diameter, approximately equal to `3.1415926535897932`.

- `infinity` & `-infinity`

  - : An infinite value, used to indicate the largest/smallest possible value.

- `NaN`

>   - : A value representing "Not a Number". This constant case-sensitive.

Reviewer: ```suggestion
  - : A value representing "Not a Number". This constant is case-sensitive.
```
